cluser_validation_checklist.md

```diff
@@ -0,0 +1,87 @@
+# Cluster Validation Checklist
+## Check general cluster status
+
+ - [ ] Review installation log
+   Check for any glaring issues, cluster admin credentials can be pulled from here as well.
+   Install logs can be found on the install node, inside of the install directory
+   ```shell
+   $ cat <install_dir>/.openshift_install.log
+   ```
+
+ - [ ] Verify that image pull source is what is expected
+   Check that the source of your openshift images is as expected.
+   As we are not in a disconnected environment quay.io and the internal cluster registry.
+   ```shell
+   $ oc adm node-logs <node_name> -u crio
+   ```
+
+ - [ ] Verify that cluster version matches what is expected
+   We'll be running the latest generally available version of OpenShift.
+   At the time of writing that is 4.10.
+   ```shell
+   $ oc get cluserversion
+   ```
+
+ - [ ] Verify that the cluster is on the correct update channel
+   Currently we're looking at the stable update channel for 4.10
+   ```shell
+   $ oc get cluserversion -o jsonpath='{.items[0].spec}{"\n"}'
+   ```
+
+ - [ ] Check for any available cluster updates
```

Comment:
Not clear that we would do this step unless someone has already determined we want to be on the updated version.

Reply:
This will give us the latest version in our update channel, in our case 4.10-stable. This will only check for minor revisions and I feel that with a brand new install it's the safest and best time to upgrade to the latest version inside of your selected update channel.